javascripts/discourse/widgets/layouts-category-list.js

```diff
@@ -174,7 +186,11 @@ export default layouts.createLayoutsWidget('category-list', {
       this.isParentOfCurrent(category) ||
       this.isGrandparentOfCurrent(category);
     const hasChildren = children.length > 0;
-    const showChildren = current && hasChildren && !hideChildren[category.id];
+    const childrenAlwaysShown =
```

Comment:
Do we want children to always be shown when the default state is expanded? Shouldn't we let the user still collapse them after initial load?

Reply:
Yes, that's still the case. Sorry I think this is just poor variable naming on my part. I've updated it to be more clear.